test/relayclient/RelayClient.test.ts

```diff
@@ -156,13 +162,6 @@ contract('RelayClient', function (accounts) {
       assert.equal(forwarder, forwarderAddress)
     })
 
-    it('should throw an exception if recipient does not return address for getTrustedForwarder', async function () {
-      const badRecipient = await TestVersions.new()
-      const noForwarderDetails = Object.assign({}, options, { forwarder: undefined, to: badRecipient.address })
-      await expect(relayClient.resolveForwarder(noForwarderDetails))
-        .to.eventually.be.rejectedWith('No forwarder address configured and no getTrustedForwarder in target contract')
-    })
-
     it('should throw an exception if passed forwarder is not trusted', async function () {
```

Comment:
Don't you still have an error thrown for a similar scenario? There is no new test for the `paymaster has no trustedForwarder` error, I think.

Reply:
Yes.
I want to add a separate PR to validate call: is optionally (and defaults to true) to check all request params and values.
When disabled, only solidity checks are done, and the call is faster.

Reply:
actually, misconfigured paymaster (zero forwarder or zero relayhub) can easily be checked in resolveConfigurationGsn, but it doesn't belong to this PR.